README.md

```diff
@@ -0,0 +1,26 @@
+## Command Line Interface Calculator
+
+### based on Shunting yard algorithm
+
+#### SDK: OpenJDK 21.0.2
+
+To use program:
+
+* cd src
+* Compile:```javac -d out Main.java```
+* Run: ```java -cp ./out Main.java```
+
+---
+
+To use tests:
+
+* cd src
+* Compile:```javac -d out -cp "junit-platform-console-standalone-1.9.3.jar;." test\CalculateTest.java test\ShuntingYardTest.java```
```

Comment:
Try to use mask for source file, in order to get rid of enumerating each file separately
As shown here - https://www.baeldung.com/java-compile-multiple-files